include/exadg/incompressible_navier_stokes/time_integration/driver_steady_problems.cpp

```diff
@@ -163,7 +164,7 @@ DriverSteadyProblems<dim, Number>::do_solve(double const time, bool unsteady_pro
     if(this->param.right_hand_side)
       pde_operator->evaluate_add_body_force_term(rhs, time);
 
-    // Newton solver
+    // Newton solver with viscosity update
```

Comment:
This comment "with viscosity update" seems to be irritating. It looks like the Newton solver will always do a viscosity update. However, it will only be done if the nonlinear viscosity term is treated implicitly, which is only one reason for the if-statement `this->param.nonlinear_problem_has_to_be_solved()` above to become `true`.

So I think I would actually skip this here.